docs/spec/spec.md

```diff
@@ -1548,13 +1548,13 @@ service on new graphql:Listener(4000) {
 
 #### 9.1.7 Introspection Configurations
 
-The `introspectionEnabled` field is used to enable or disable the GraphQL introspection query support. If the introspection query support is disabled, the GraphQL service won't allow the execution of the `__schema` and the `__type` introspection queries. However, the `__typename` introspection will work even if the introspection query support is disabled.
+The `introspection` field is used to enable or disable the GraphQL introspection query support. If the introspection query support is disabled, the GraphQL service won't allow the execution of the `__schema` and the `__type` introspection queries. However, the `__typename` introspection will work even if the introspection query support is disabled.
```

Comment:
Shouldn't we mention that it is enabled by default?

Reply:
It's included in the `Note` below the example.

Reply:
Let's add it to the description and keep the note stating the recommendation to disable it in production.

Reply:
Wording should be something like this:

```
... introspection will work even if the introspection query support is disabled. By default, introspection is enabled for Ballerina GraphQL services.
```

And the note:

```
**Note:** It is recommended to disable introspection in production environments.
```